test/users.js

```diff
@@ -103,6 +103,38 @@ describe('Users', function() {
       });
   });
 
+  it('should verify a user password created with former credential package', function(done) {
+    var req = apos.tasks.getReq();
+    var user = apos.users.newInstance();
+
+    user.firstName = 'Old';
+    user.lastName = 'User';
+    user.title = 'Old User';
+    user.username = 'olduser';
+    user.password = 'passwordThatThroughOldCredentialPackageHashing';
```

Comment:
`passwordThatThroughOldCredentialPackageHashing` -> Went _Through_, forgot that.. but we're in a test, and in A2, so I guess it's ok